besu/src/main/java/org/hyperledger/besu/services/BesuPluginContextImpl.java

```diff
@@ -89,6 +90,10 @@ public void registerPlugins(final Path pluginsDir) {
       try {
         plugin.register(this);
```

Comment:
If we do plugin versions I think we should extract it prior to the plugin registration and make version information a pre-requisite and not register it if there is no plugin version information.

Reply:
Alternately if we don't require a version can we add it to the version list with <unknown version> instead of discarding it?

Reply:
I am inclining towards <Unknown Version> approach.